core/src/main/java/greencity/service/impl/CloudStorageService.java

```diff
@@ -64,4 +67,23 @@ private URL write(final BlobInfo blobInfo, final MultipartFile multipartFile) th
     private URL getURL(BlobInfo blobInfo) throws MalformedURLException {
         return new URL(staticUrl + blobInfo.getBucket() + "/" + blobInfo.getName());
     }
+
+    /**
+     * Convert string to MultipartFile.
+     *
+     * @return MultipartFile.
+     **/
+    public MultipartFile convertToMultipartImage(String image) {
+        String imageToConvert = image.substring(image.indexOf(',') + 1);
+        File tempFile = new File("tempImage.jpg");
+        byte[] imageByte = decodeBase64(imageToConvert);
+        ByteArrayInputStream bis = new ByteArrayInputStream(imageByte);
+        try {
+            BufferedImage bufferedImage = ImageIO.read(bis);
+            ImageIO.write(bufferedImage, "png", tempFile);
+            return new MockMultipartFile(tempFile.getPath(), new FileInputStream(tempFile));
```

Comment:
Please do not use classes intended for testing in main code. 

Please also change the scope of `org.springframework:spring-test` to `test `in pom.xml